### PR TITLE
translation into brazilian portuguese

### PR DESCRIPTION
**Pull request by luizcmarin**
| New feature?  | ✔️

### CODE REVIEW

File: docs/guide/en/README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 - [Hydrator attributes](hydrator-attributes.md)
 - [Request input](request-input.md)
-- [Parameters' resolvers](parameters-resolvers.md)
+- [parameters resolvers](parameters-resolvers.md)
```
Comment (samdark) [inline docs/guide/en/README.md:5]: It was good as is.

File: docs/guide/en/parameters-resolvers.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# Parameters' resolvers
+# parameters resolvers
```
Comment (samdark) [inline docs/guide/en/parameters-resolvers.md:1]: It was good as is.

File: docs/guide/en/parameters-resolvers.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
 This package offers multiple
-[custom implementations of parameters' resolver](https://github.com/yiisoft/middleware-dispatcher?tab=readme-ov-file#creating-your-own-implementation-of-parameters-resolver) 
+[custom implementations of parameters resolver](https://github.com/yiisoft/middleware-dispatcher?tab=readme-ov-file#creating-your-own-implementation-of-parameters-resolver)
```
Comment (samdark) [inline docs/guide/en/parameters-resolvers.md:4]: It was good as is.

File: docs/guide/en/parameters-resolvers.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ final class PostController
 
 ### Adding to middleware dispatcher
 
-An example of applying parameters' resolver to middleware dispatcher:
+An example of applying parameters resolver to middleware dispatcher:
```
Comment (samdark) [inline docs/guide/en/parameters-resolvers.md:60]: It was good as is.

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by luizcmarin** [comment]
> https://github.com/yiisoft/input-http/actions/runs/8692486186/job/23837219643?pr=37

I didn't understand. I don't see the problem here. The syntax is similar to other lines in the same file.

**Review by samdark — APPROVED** [review]

**Pull request merged**